utils.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2021 Open Mobile Platform LLC.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#include <syslog.h>
+#include <cstring>
+#include <iostream>
+
+#include "geckocamera-utils.h"
+
+using namespace std;
+
+namespace gecko {
+namespace camera {
+namespace internal {
+
+static bool logInitialized;
```

Comment:
Some default value would be nice here.

Reply:
This is a static variable and is always set to zero, but I agree that it looks better with an initializer.